### PR TITLE
drop symfony 4.4 support

### PR DESCRIPTION
**Pull request by jrushlow**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -6,17 +6,18 @@
...
     "require-dev": {
         "doctrine/orm": "^2.7",
         "doctrine/persistence": "^2.0",
-        "symfony/framework-bundle": "^4.4 | ^5.0 | ^6.0",
-        "symfony/phpunit-bridge": "^5.0 | ^6.0",
+        "symfony/framework-bundle": "^5.4 | ^6.0",
+        "symfony/phpunit-bridge": "^5.4 | ^6.0",
         "vimeo/psalm": "^4.3"
     },
     "conflict": {
```
Comment (jrushlow) [inline composer.json:23]: We should be able to remove this conflict now if i'm not mistaken

Comment (bocharsky-bw) [reply composer.json:23]: Hm, according to the version constraint we have so far: `^4.4 | ^5.0 | ^6.0"` - it sounds like we could drop this even before this PR as `<4.4` is not an option anymore, unless I'm missing something

**Comment by bocharsky-bw** [comment]
Well, tests are green here, but do we really want to drop 4.4 support? If it is worth almost nothing to keep its support - probably better to keep it. But if it causes some issues or gets hard to maintain - I'm fine to remove it

**Review by weaverryan — DISMISSED** [review]
good, other than the one comment

**Pull request merged**